toolkit/scripts/imggen.mk

```diff
@@ -11,7 +11,11 @@ assets_files             = $(shell find $(assets_dir))
 imggen_local_repo        = $(MANIFESTS_DIR)/image/local.repo
 imagefetcher_local_repo  = $(MANIFESTS_DIR)/package/local.repo
 imagefetcher_cloned_repo = $(MANIFESTS_DIR)/package/fetcher.repo
+ifeq ($(shell uname -m),aarch64)
```

Comment:
Can reference `$(build_arch)` variable here. Applies to below as well.

Reply:
updated.